vj4/ui/templates/contest_main.html

```diff
@@ -48,9 +48,9 @@ <h2 class="status_title">{{ _(title) }}</h2>
 <div class="row">
   <div class="medium-9 columns">
   {% for tdoc in tdocs %}
-  {% if handler.is_live(tdoc) %}
+  {% if handler.is_ongoing(tdoc) %}
     {{ render_important_contest('live', 'Live...', tdoc) }}
-  {% elif handler.is_ready(tdoc) %}
+  {% elif handler.is_upcoming(tdoc) %}
```

Comment:
???

Reply:
"Ready" is not a correct usage of English, renamed to "Upcoming".